### DISCUSSION TITLE
WPILib Beta 3

### DISCUSSION THREAD

**Discussion by PeterJohnson** [wpilibsuite/2025Beta#79] [Announcements]
WPILib Beta 3 has been released!

https://github.com/wpilibsuite/allwpilib/releases/tag/v2025.1.1-beta-3

**Comment by luniish** [comment]
Hello, I have problems downloading the file. 

![image](https://github.com/user-attachments/assets/b34ebdfd-65ae-472a-90d5-8cb57f09d002)

**Reply by sciencewhiz** [reply]
Are you using an internet connection that is blocking the download?
